### DISCUSSION TITLE
Why is rendered content wrapped with <article>...</article>?

### DISCUSSION THREAD

**Discussion by paulrudy** [markdoc/markdoc#347] [Q&A] [answered]
I notice that Markdoc wraps `<article>`...`</article>` around rendered content. I'm switching from using MDX, which didn't behave that way. I'm curious what the reasoning is for including the wrapping `<article>`...`</article>` tags, and whether that might made optional in future releases.

**Comment by rpaul-stripe** [comment]
We chose this as the default behavior because we wanted the top-level transform to return a single render tree node. It can easily be changed by overriding the `document` node in your Markdoc configuration. If you want `Markdoc.transform` to return an array that isn't wrapped in a tag (which Markdoc's React renderer will turn into a `Fragment`), you can use the following configuration:

```
import Markdoc from '@markdoc/markdoc';

const config = {
  nodes: {
    document: {...Markdoc.nodes.document, render: null}
  }
};

const doc = Markdoc.parse(example);
const output = Markdoc.transform(doc, config);
```

**Reply by paulrudy** [reply]
Thanks, I appreciate the explanation and the configuration!

**Comment by skix123** [comment, marked as answer]
2025 update, use:
```
document: {...Markdoc.nodes.document, render: ''}
```
Note the `render: ''` (empty string) prop, `null` is not a valid value (https://markdoc.dev/docs/nodes#options)
